app/components/preprint-status-banner/template.hbs

```diff
@@ -10,7 +10,15 @@
             </span>
         </div>
         <div class="recent-activity">
-            <a href={{creatorProfile}}>{{creatorName}}</a> {{t recentActivityLanguage}} {{moment-format submission.dateLastTransitioned "MMMM DD, YYYY"}}
+            {{#if loadingActions}}
+                {{loading-indicator classes='text-center m-b-lg p-v-md'}}
```

Comment:
The extra padding isn't needed when loading:
![screen shot 2017-10-09 at 11 12 03 am](https://user-images.githubusercontent.com/7131985/31345094-b53686ce-ace2-11e7-9aa9-ef3ac39b860f.png)
![screen shot 2017-10-09 at 11 12 38 am](https://user-images.githubusercontent.com/7131985/31345117-ca8bfa22-ace2-11e7-975a-e8d5dec9ce91.png)